src/application.coffee

```diff
@@ -332,6 +363,122 @@ updateStatus =
 	forceNext: false
 	intervalHandle: null
 
+updateStrategies =
+	'normal-update': (localApp, app, needsDownload, force, timeout) ->
```

Comment:
It'd be nice to name this based on its behaviour like for the others, that way the decision of what's "normal" is within the api and can be changed more easily if need be.  Not to mention it acts as documentation for this bit of code.